examples/advanced/sklearn-svm/jobs/sklearn_svm_base/app/custom/svm_learner.py

```diff
@@ -79,7 +90,16 @@ def train(self, curr_round: int, global_param: Optional[dict], fl_ctx: FLContext
     def validate(self, curr_round: int, global_param: Optional[dict], fl_ctx: FLContext) -> Tuple[dict, dict]:
         # local validation with global support vectors
         # fit a standalone SVM with the global support vectors
-        svm_global = SVC(kernel=self.kernel)
+        if self.backend == "sklearn":
+            from sklearn.svm import SVC
```

Comment:
It might be better to use the `optional_import` once as [here](https://github.com/NVIDIA/NVFlare/blob/793dcaefe23123c290edca21f400ed03d83c381c/nvflare/app_opt/xgboost/histogram_based/controller.py#L87) instead of importing twice in the same function

Reply:
updated

Reply:
updated code LGTM.